independent-projects/tools/devtools-common/src/test/resources/other-recipes.yaml

```diff
@@ -0,0 +1,913 @@
+      artifactId: jakarta.enterprise.cdi-api
+      newVersion: 2.x
+
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: javax.decorator
+      newPackageName: jakarta.decorator
+      recursive: true
+
+  # Remove Javax API(x)
+  - org.openrewrite.maven.RemoveDependency:
+      groupId: javax.enterprise
+      artifactId: cdi-api
+
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JavaxEjbToJakartaEjb
+displayName: Migrate deprecated `javax.ejb` packages to `jakarta.ejb`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+recipeList:
+#  - org.openrewrite.maven.AddDependency:
+#      groupId: jakarta.ejb
+#      artifactId: jakarta.ejb-api
+#      version: 4.x
+#      onlyIfUsing: javax.ejb.+
+
+  # Upgrade the dependency to use the jakarta namespace if an older version already exists.
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: jakarta.ejb
+      artifactId: jakarta.ejb-api
+      newVersion: 4.x
+
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: javax.ejb
+      newPackageName: jakarta.ejb
+      recursive: true
+
+  # Remove Javax API(x)
+  - org.openrewrite.maven.RemoveDependency:
+      groupId: javax.ejb
+      artifactId: javax.ejb-api
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JavaxElToJakartaEl
+displayName: Migrate deprecated `javax.el` packages to `jakarta.el`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+recipeList:
+#  - org.openrewrite.maven.AddDependency:
+#      groupId: jakarta.el
+#      artifactId: jakarta.el-api
+#      version: 4.x
+#      onlyIfUsing: javax.el.*
+
+  # Upgrade the dependency to use the jakarta namespace if an older version already exists.
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: jakarta.el
+      artifactId: jakarta.el-api
+      newVersion: 4.x
+
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: javax.el
+      newPackageName: jakarta.el
+      recursive: true
+
+  # Remove Javax API(x)
+  - org.openrewrite.maven.RemoveDependency:
+      groupId: javax.el
+      artifactId: javax.el-api
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JavaxEnterpriseToJakartaEnterprise
+displayName: Migrate deprecated `javax.enterprise` packages to `jakarta.enterprise`
+description: Java EE has been rebranded to Jakarta EE, necessitating a package relocation.
+recipeList:
+#  - org.openrewrite.maven.AddDependency:
+#      groupId: jakarta.enterprise
+#      artifactId: jakarta.enterprise.cdi-api
+#      version: 3.0.1
+#      onlyIfUsing: javax.enterprise.+
+
+  # Upgrade the dependency to use the jakarta namespace if an older version already exists.
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: jakarta.enterprise
+      artifactId: jakarta.enterprise.cdi-api
+      newVersion: 3.0.1
```

Comment:
these version values should be quoted, i.e. "3.0.1" to avoid yaml double conversion losses.

Reply:
Yes https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Jakarta.20migration/near/311189149

Reply:
```
jbang https://gist.github.com/maxandersen/7c253c7c30ce3fb959f00aa412bf98d4
[jbang] Building jar...
[{"org.openrewrite.maven.ChangePropertyValue":null,"key":"antlr.version","newValue":4.1},{"org.openrewrite.maven.ChangePropertyValue":null,"key":"version.jpa","newValue":"3.0.0"}]
DOUBLE
```

so the issue is that 4.10 gets parsed into a double which shortens to 4.1 when converted tostring.

"4.10" ensures its treated as an actual string not a number.

Reply:
ok thanks, I'll try to find the setting in snakeyaml for this

Reply:
it seems snake yaml is smart enough:
```yaml
- org.openrewrite.maven.ChangePropertyValue: {
    key: quarkus.platform.version,
    newValue: '3.0'
  }
- org.openrewrite.maven.ChangePropertyValue: {
    key: quarkus.version,
    newValue: 3.0.0.Alpha1
  }
```

Reply:
@maxandersen 👆